README.md

```diff
@@ -0,0 +1,31 @@
+What is this?
+=============
+
+Frotz activity is a simple front end to the classic interactive fiction interpreter Frotz on the XO laptop.  Frotz activity is licensed under the GNU General Public License version 3 (see the file COPYING for details). 
```

Comment:
No longer for the XO laptop.

Reply:
Details of the licensing are actually in frotz.py, not in COPYING.  COPYING does contain details, but not the copyright and license.  If in doubt as to the best thing to say; what is in frotz.py header is exactly what should be said.

Reply:
Done in c89585f